Index: os-pylint-twisted/tests/reactor.txt
===================================================================
--- os-pylint-twisted.orig/tests/reactor.txt
+++ os-pylint-twisted/tests/reactor.txt
@@ -1,3 +1,3 @@
 missing-module-docstring:1:0:None:None::Missing module docstring:HIGH
-missing-function-docstring:5:0:7:18:call_when_running:Missing function or method docstring:HIGH
+missing-function-docstring:5:0:5:21:call_when_running:Missing function or method docstring:HIGH
 no-member:6:4:6:22:call_when_running:Instance of 'EPollReactor' has no 'attr_error' member:INFERENCE
Index: os-pylint-twisted/tests/test_defer_inlinecallbacks.py
===================================================================
--- os-pylint-twisted.orig/tests/test_defer_inlinecallbacks.py
+++ os-pylint-twisted/tests/test_defer_inlinecallbacks.py
@@ -25,6 +25,10 @@ def foo():
             Message(
                 msg_id="does-not-produce-generator",
                 node=function_node,
+                line=4,
+                end_line=4,
+                col_offset=0,
+                end_col_offset=7,
             ),
         ):
             self.checker.leave_functiondef(function_node)
@@ -71,6 +75,10 @@ def foo():
             Message(
                 msg_id="does-not-produce-generator",
                 node=function_node,
+                line=4,
+                end_line=4,
+                col_offset=0,
+                end_col_offset=7,
             ),
         ):
             self.checker.leave_functiondef(function_node)
Index: os-pylint-twisted/tests/test_defer_returnvalue.py
===================================================================
--- os-pylint-twisted.orig/tests/test_defer_returnvalue.py
+++ os-pylint-twisted/tests/test_defer_returnvalue.py
@@ -22,6 +22,10 @@ defer.returnValue(True)
             Message(
                 msg_id="legacy-return",
                 node=call_node,
+                line=2,
+                end_line=2,
+                col_offset=0,
+                end_col_offset=23,
             ),
         ):
             self.checker.visit_call(call_node)
@@ -36,6 +40,10 @@ returnValue(True)
             Message(
                 msg_id="legacy-return",
                 node=call_node,
+                line=2,
+                end_line=2,
+                col_offset=0,
+                end_col_offset=17,
             ),
         ):
             self.checker.visit_call(call_node)
